.github/workflows/node.js.yml

```diff
@@ -47,3 +47,21 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - run: npx @adobe/sizewatcher
+  semantic-release:
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: ${{  !contains(github.event.head_commit.message, '[ci skip]') && github.ref == 'refs/heads/master' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: Use Node.js 12.19
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.19'
+      - run: npm install
+      - run: npm run build
+      - run: npm run semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.ADOBE_BOT_GITHUB_TOKEN }}
```

Comment:
We don't have access to the tokens secret - If they got rotated or are outdated, you'll have to ask the open-source team to update them. Otherwise the automated release won't work.

Reply:
(but otherwise the config looks good)

Reply:
Good catch, thanks @tmathern! I checked the [old action](https://github.com/adobe/aem-upload/blob/ce3416912cf38ddfa71edda8534f3dd0b7fb04b4/.github/workflows/release.yml#L33) and it was using `secrets.GITHUB_TOKEN`, so I updated the new action to match. The "old" action worked with this value - do you think it will work with semantic release?

Reply:
I don't think so (but check, you never know). We had to bug the open source people so the token gets updated.